src/Error/Middleware/ErrorHandlerMiddleware.php

```diff
@@ -139,7 +140,17 @@ protected function getRenderer($exception)
         if (is_string($this->exceptionRenderer)) {
             $class = App::className($this->exceptionRenderer, 'Error');
             if (!$class) {
-                throw new Exception("The '{$this->exceptionRenderer}' renderer class could not be found.");
+                throw new Exception(sprintf(
+                    "The '%s' renderer class could not be found.",
+                    $this->exceptionRenderer
+                ));
+            }
+            if (!$class instanceof ExceptionRendererInterface) {
```

Comment:
This is not fully BC since it will fail for all existing custom exception renderers which don't extend the core `ExceptionRender`.